core/dbt/contracts/graph/manifest.py

```diff
@@ -616,7 +616,8 @@ class Manifest(MacroMethods, DataClassMessagePackMixin, dbtClassMixin):
     flat_graph: Dict[str, Any] = field(default_factory=dict)
     state_check: ManifestStateCheck = field(default_factory=ManifestStateCheck)
     source_patches: MutableMapping[SourceKey, SourcePatch] = field(default_factory=dict)
-    disabled: MutableMapping[str, List[CompileResultNode]] = field(default_factory=dict)
+    # TODO: not sure this is the right move here?
+    disabled: MutableMapping[str, List[GraphMemberNode]] = field(default_factory=dict)
```

Comment:
This also seems off but exposures/metrics are not compiled.

Reply:
Exposures aren't compiled but metrics are.  So "GraphMemberNode" isn't quite accurate.

Reply:
Well, it's IS true that it "participates" in the graph. So its fine.